android/app/src/main/java/com/microsoft/codepush/react/CodePush.java

```diff
@@ -56,185 +56,43 @@
         return new ArrayList<>();
     }
 
-    /**
```

Comment:
These methods should be deprecated since new version of CodePush. If they will be dropped now then breaking change for API will be introduced and customers will have to change their CodePush initialization code to make it work.